sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/literals.scala

```diff
@@ -499,7 +499,7 @@ case class Literal (value: Any, dataType: DataType) extends LeafExpression {
         }
       val structFields: Array[String] =
         structNames.zip(structValues).map {
-          kv => s"${kv._1}, ${kv._2}"
+          kv => s"'${kv._1}', ${kv._2}"
```

Comment:
Why doesn't the end-to-end test fail before the code change?

Reply:
It was accidentally not getting exercised, it turns out :) but now it is.